sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/api/ApiPathPredicateItem.java

```diff
@@ -15,9 +15,10 @@
  */
 package com.alibaba.csp.sentinel.adapter.gateway.common.api;
 
+import com.alibaba.csp.sentinel.adapter.gateway.common.SentinelGatewayConstants;
+
```

Comment:
Could you please remove the unrelated changes in `sentinel-api-gateway-adapter-common` module? This PR will be squashed and it should only contain changes related to config :)

Reply:
Thanks for reminding, this is a mistake, formatting the code when merging the master code. I have revert it.